db/db.go

```diff
@@ -76,623 +70,746 @@ func New(db *sql.DB) DB {
 	}
 }
 
-// Init creates the tables for storing txs if it does not exist. Multiple calls
-// of this function will only create the tables once and not return an error.
-// TODO: Decide approach for versioning database tables.
+// Init creates the tables for storing transactions if they do not already
+// exist. The tables will only be created the first time this funciton is called
+// and any future calls will not return an error.
 func (db database) Init() error {
+	// Create the lock-and-mint (UTXO -> account) table if it does not exist.
+	lockUTXOMintAccount := `CREATE TABLE IF NOT EXISTS lock_utxo_mint_account (
+		hash          VARCHAR NOT NULL PRIMARY KEY,
+		status        SMALLINT,
+		created_time  BIGINT,
+		selector      VARCHAR(255),
+		utxo_hash     VARCHAR,
+		utxo_index    BIGINT,
+		value         VARCHAR(100),
+		pubkey_script VARCHAR,
```

Comment:
Is there a reason we do not prefix these values with `utxo_` too? For example: `utxo_value` and `utxo_pubkeyscript`? (Noting this with respect to `utxo_hash` and `utxo_index`.)

Reply:
Don't think there's any reason. I've updated it now